### PR TITLE
[WIP] cob-uh final

### PR DESCRIPTION
**Pull request by fmessmer**
- HW still not complete
 - Long-Term/Robustness tests still pending

### CODE REVIEW

**Review by floweisshardt — DISMISSED** [review]

File: cob_bringup/robots/cob4-16.xml [hunk 1]
```diff
@@ -131,20 +131,20 @@
 			<arg if="$(arg sim)" name="cartesian_control" value="true"/>
 		</include>
 
-		<!--include file="$(find cob_bringup)/components/canopen_generic.launch">
+		<include file="$(find cob_bringup)/components/canopen_generic.launch">
 			<arg name="robot" value="$(arg robot)"/>
 			<arg name="component_name" value="arm_left"/>
-			<arg name="can_device" value="can3"/>
+			<arg name="can_device" value="can2"/>
```
Comment (floweisshardt) [inline cob_bringup/robots/cob4-16.xml:137]: why are can devices switched for the arms? in cob4-5 and cob4-8 we have 
- arm_right: can2
- arm_left: can3

File: cob_hardware_config/robots/cob4-16/config/arm_left_driver.yaml [hunk 2]
```diff
@@ -14,16 +14,16 @@ defaults:
 
 nodes:
   - name: arm_left_1_joint
-    id: 61
+    id: 51
```
Comment (floweisshardt) [inline cob_hardware_config/robots/cob4-16/config/arm_left_driver.yaml:17]: same comment as above: why switch arm left and right?

**Review by floweisshardt — CHANGES_REQUESTED** [review]

File: cob_hardware_config/robots/cob4-16/config/arm_left_driver.yaml (hunk 2 above)
Comment (fmessmer) [reply cob_hardware_config/robots/cob4-16/config/arm_left_driver.yaml:17]: this is how the robot hw is set up...

File: cob_bringup/robots/cob4-16.xml (hunk 1 above)
Comment (fmessmer) [reply cob_bringup/robots/cob4-16.xml:137]: this is how the robot hw is set up...

**Review by HannesBachter — APPROVED** [review]

**Review by floweisshardt — DISMISSED** [review]
ok, let's keep the hw as it is (left port on robot = left arm = can2, right port on robot = right arm = can3)

**Comment by fmessmer** [comment]
there are some more local changes on the robot, yet to be committed

**Comment by fmessmer** [comment]
all committed...everything works as expected...except grippers @HannesBachter

**Comment by floweisshardt** [comment]
I'll merge, checkout indigo_dev on robot and go through the delivery check list

**Pull request merged**